src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1499,6 +1499,7 @@ internal bool containsInvalidInputSymbols()
             return this.Nodes.OfType<Nodes.CustomNodes.Symbol>().Any(node => !node.Parameter.NameIsValid);
         }
 
+        [Obsolete("Method will be deprecated in Dynamo 3.0.")]
         private void SerializeElementResolver(XmlDocument xmlDoc)
```

Comment:
Why can't we remove this right away?

Reply:
@Astul-Betizagasti For private functions we should be able to remove any time. unlike protected and public ones